exams/signals.py

```diff
@@ -29,10 +29,10 @@ def update_exam_authorization_final_grade(sender, instance, **kwargs):  # pylint
     authorize_for_exam(mmtrack, instance.course_run)
 
 
-@receiver(post_save, sender=CachedEnrollment, dispatch_uid="update_exam_authorization_cached_enrollment")
-def update_exam_authorization_cached_enrollment(sender, instance, **kwargs):  # pylint: disable=unused-argument
+@receiver(post_save, sender=CachedCurrentGrade, dispatch_uid="update_exam_authorization_grades")
```

Comment:
i changes the title of PR and description is updated. I think we should fix every issue on exam authorization triggering here. To avoid complexity because code is already merge and think it has some issues in both signals and command